docs/requirements/stakeholder/index.rst

```diff
@@ -264,9 +265,91 @@ Dependability
    * ECC Memory
    * Software Lockstep
    * Power management integrated circuit (PMIC), external watchdog and voltage monitoring
-   * Safe switch from engineering for field mode and back
+   * Safe switch from engineering to field mode and back
 
-   Note: This is part of 0.5 release and therefore can only support ASIL_B. Goal is ASIL_D.
+
+.. stkh_req:: SW-platform error reaction
+   :id: stkh_req__dependability__error_reaction
+   :reqtype: Functional
+   :security: NO
+   :safety: ASIL_B
+   :rationale: Safety relies on error reporting, e.g. to be able to reach a safe system state.
```

Comment:
A hanging system will not need to report an error. The external watchdog can identify a missed deadline and trigger the safe state which is also a safe system state.

```suggestion
   :rationale: Safety relies on error reporting, e.g. to preserve a safe system state.
```